scripts/azurepowershellscript.ps1

```diff
@@ -0,0 +1,20 @@
+$resourceGroupName="myResourceGroup"
+$location = "East US"
+$ehubName="myEventHub"
+$namespaceName="myNamespace2023"
+$functionAppName = "AzureFunctionTriggerApp2023"
+$runtime = "powershell"
+$storageAccountName = "testdatastorage2023"
+$accountName = "mydb2023"
+$databaseName = "sql-db-2023"
+New-AzResourceGroup –Name $resourceGroupName –Location $location
+New-AzEventHubNamespace -ResourceGroupName $resourceGroupName -Name $namespaceName -Location $location
+New-AzEventHub -ResourceGroupName $resourceGroupName -NamespaceName $namespaceName -EventHubName $ehubName
+New-AzStorageAccount -ResourceGroupName $resourceGroupName -Name $storageAccountName -Location $location -SkuName Standard_LRS
+New-AzAppServicePlan -ResourceGroupName $resourceGroupName -Name "$functionAppName-plan" -Location $location -Tier Free
+New-AzFunctionApp -Name $functionAppName -ResourceGroupName $resourceGroupName -StorageAccount $storageAccountName -Runtime PowerShell -FunctionsVersion 4 -Location $location
+New-AzCosmosDBAccount -ResourceGroupName $resourceGroupName -Name $accountName -Location $location -DefaultConsistencyLevel "Session"
+$database = New-AzCosmosDBSqlDatabase -AccountName $accountName -ResourceGroupName $resourceGroupName -Name $databaseName
+$account = Get-AzCosmosDBAccount -ResourceGroupName $resourceGroupName -Name $accountName

```

Comment:
What is the use of these two variables?

Reply:
Created a SQLDatabase inside CosmoDB Account and stored it in $database variable. Similarly stored the CosmosDBAccount details inside the $account variable. The $account variable is used in the below connection string generation. Should we remove these?